dependency/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: retrieve
```

Comment:
```suggestion
.PHONY: test retrieve
```

Reply:
Running `make test version=72.1 tarballPath=<path to tarball>` does not work, even after I run add this line locally I get:

make: *** [test] Error 1